handler/oauth2/flow_authorize_implicit_test.go

```diff
@@ -138,3 +131,49 @@ func TestAuthorizeImplicit_EndpointHandler(t *testing.T) {
 		})
 	}
 }
+func makeAuthorizeImplicitGrantTypeHandler(ctrl *gomock.Controller) (AuthorizeImplicitGrantTypeHandler,
```

Comment:
We should have a test which checks if it is allowed to set the response mode to query while asking to perform the implicit flow.

Reply:
The validation happens outside of the handler so we can't test here. Added the scenario in integration tests.